assets/src/styles/blocks/Issues.scss

```diff
@@ -0,0 +1,107 @@
+.issues-block-pattern {
```

Comment:
Ideally, we should avoid adding custom classes to the patterns, because the second it gets added to a page it's basically just a block... We can style the root block (for example the WP core `Columns` block, I created a [ticket](https://jira.greenpeace.org/browse/PLANET-6683) to get design recommendations for it), but then it will be styled for all occurrences of said block, not just the one used in the pattern. It's actually better for consistency too.
There are some native ways to style the blocks that are used in the pattern:
- You can usually define color and background color in the attributes when initialising a block (such as `Media & Text`).
- For spacing, it's nice to use the `Spacer` block or [Bootstrap classes](https://getbootstrap.com/docs/5.0/utilities/spacing/) that we already use in other places.

It is less customisable than adding a custom class, so it means sometimes we'll have to compromise, but I think it makes more sense for patterns 🙂